tests/pfcwd/test_pfc_config.py

```diff
@@ -320,7 +324,7 @@ def test_default_cfg_after_load_mg(self, duthosts, enum_rand_one_per_hwsku_front
             None
         """
         duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
-        config_reload(duthost, config_source='minigraph')
+        config_reload(duthost, config_source='minigraph', safe_reload=True)
```

Comment:
Also fyi @ZhaohuiS in here since we're doing a minigraph config_reload. It's unavoidable that we're going to experience config_db_check fail. I've added a safe_reload=True here as suggested by @yejianquan to keep the testbed stable